tests/core/utilities/test_stability.py

```diff
@@ -0,0 +1,50 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+from flash.core.utilities.stability import beta
+
+
+@beta()
+class _BetaType:
+    pass
+
+
+@beta("_BetaTypeCustomMessage is currently in Beta.")
+class _BetaTypeCustomMessage:
+    pass
```

Comment:
Do you also want to test to ensure that when a beta class method is called - no warning is raised? Since I guess, we only raise a warning when a class is instantiated.

Also, a question - what if we have a static method in a beta class? And if I just call it: `PointCloudObjectDetection.some_static_method()`, will it raise a warning? I guess not, right?

Reply:
>  And if I just call it: PointCloudObjectDetection.some_static_method(), will it raise a warning? I guess not, right?

That's right. My thinking is that we can manually annotate classmethods if needed, but we could later extend this to do that for us automatically 😃  I'll keep the test simple for now until we have used the decorator in a few more places and know what we want from it.

Reply:
> My thinking is that we can manually annotate classmethods if needed, but we could later extend this to do that for us automatically

Makes sense to me! :)

> I'll keep the test simple for now until we have used the decorator in a few more places and know what we want from it.

Sure! For now, this looks great. I agree, once we use it more, we can come back to the tests and extend them if required.